go.mod

```diff
@@ -42,19 +42,18 @@ require (
 	github.com/pingcap/sysutil v0.0.0-20200206130906-2bfa6dc40bcd
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.0.0
-	github.com/sirupsen/logrus v1.2.0
-	github.com/spf13/cobra v0.0.3
-	github.com/spf13/pflag v1.0.1
+	github.com/sirupsen/logrus v1.4.2
```

Comment:
It looks like the `logrus` upgrade caused CI to fail.

Reply:
Yes, I downgrade logrus now.